src/js/themes/hpe.js

```diff
@@ -3114,6 +3112,23 @@ const buildTheme = (tokens, flags) => {
     thumbsRating: {
       like: { color: 'background-selected-primary-strong' },
```

Comment:
if we are declaring `color` below in the icons block do we need these lines? @Sulaymon333